Kinvey/Kinvey/DataStore.swift

```diff
@@ -186,8 +186,17 @@ open class DataStore<T: Persistable> where T: NSObject {
      - returns: A `Request` instance which will allow cancel the request later
      */
     @discardableResult
-    open func find(_ id: String, readPolicy: ReadPolicy? = nil, completionHandler: @escaping ObjectCompletionHandler) -> Request {
-        return find(id, readPolicy: readPolicy) { (result: Result<T, Swift.Error>) in
+    open func find(
+        _ id: String,
+        readPolicy: ReadPolicy? = nil,
```

Comment:
We should deprecate the `readPolicy` parameter and make it part of the `requestConfig`.